A40-csds-support.md

```diff
@@ -161,20 +122,6 @@ which includes the version, timestamp, and the reason of the rejected update
 (see [proto
 definition](https://github.com/envoyproxy/envoy/blob/a4024a578b3f2611fe26229f5d0de99eb0c56895/api/envoy/admin/v3/config_dump.proto#L72)):
```

Comment:
The trailing `:` should now be a `.` instead.

Reply:
Done.